henry/commands/analyze.py

```diff
@@ -16,16 +17,19 @@ def run(cls, user_input: fetcher.Input):
             result = analyze.explores(
                 model=user_input.model, explore=user_input.explore
             )
+        else:
+          raise ValueError(
+              "Please specify one of 'projects', 'models' or 'explores'")
```

Comment:
There may be a more elegant way to handle this, but it felt sensible to catch these